jekyll/_ccie/aws.md

```diff
@@ -1,69 +1,76 @@
+- Adequately power the NAT instance for heavy network operations.  Depending on the specifics of your deployment, it is possible for NAT instances to become constrained by highly parallel builds using Docker and external network resources.  A NAT that is inadequate could cause slowness in network and cache operations.
+  - If you are integrating with https://github.com/, ensure that your network access control list (ACL) whitelists github.com webhooks.  When integrating with GitHub, either set up CircleCI in a public subnet, or set up a public load balancer to forward github.com traffic.
 
-1. Download the setup repo: `git clone https://github.com/circleci/enterprise-setup ccie && cd ccie`
+## Steps for Installation
+
+This procedure uses Terraform to automate infrastructure provisioning and management of the CircleCI services component and builders cluster.
+
+1. Download the set up repository: `git clone https://github.com/circleci/enterprise-setup ccie && cd ccie`
 1. Open `terraform.tfvars` in your favorite text editor and input the
 prerequisites stated above in the required fields.
 1. Launch the Terraform stack:
-	- **Linux/Windows:** Please install [Terraform](https://www.terraform.io/downloads.html) yourself on Linux/Windows, then run `terraform apply` from the repo root.
-	- **OSX:** You can just run `bin/terraform apply` and follow the prompts.
+	- **Linux or Windows:** Run the `terraform apply` command from the repo root.
+	- **OSX:** Run the `bin/terraform apply` command and follow the prompts.
+1. Go to the URL output from Terraform and click Get Started.
+1. Trust the temporary SSL cert.
+1. Use a new or exisiting self-signed certificate.
+1. Upload your CircleCI license.
+1. Secure the console with a password.
+1. Complete the settings form.
+1. Save and start the CircleCI application. The System Console Dashboard appears with an indication that the app has started as shown in the following image.
 
-## Configure CircleCI Enterprise
+<img src="{{site.baseurl}}/assets/img/docs/started.png" alt="Look For 'Open'" width="150" style="margin: 10px; margin-left: 200px">
 
-1. Go To the URL output from Terraform
-1. Click "Get Started"
-1.  Trust the temporary SSL cert
-1. Use a self-signed certificate ( or if you already have one feel free to use it here )
-1. Upload your license
-1. Secure the console with a password
-1. Fill out the settings form
-1. Save and start the app (all other default settings are fine for now). You will be redirected to the System Console Dashboard,
-where you should soon see an indication that the app has started like below.
+## Next Steps for Getting Started
 
-<img src="{{site.baseurl}}/assets/img/docs/started.png" alt="Look For 'Open'" width="150" style="margin: 10px; margin-left: 200px">
+1. Click the Open link in the dashboard to go to the CircleCI Enterprise app. The Starting page for a few minutes to as the CircleCI application is booting up, then automatically redirects to the homepage.
```

Comment:
Can we add a note here that the first user to access the CircleCI app becomes the admin?